plugins/inputs/win_wmi/README.md

```diff
@@ -0,0 +1,280 @@
+# Windows Management Instrumentation Input Plugin
+
+This document presents the input plugin to read WMI classes on Windows
+operating systems. With the win_wmi plugin, an administrator is enabled to
+capture and filter virtually any configuration or metric value exposed through
+the Windows Management Instrumentation
+([WMI](https://learn.microsoft.com/en-us/windows/win32/wmisdk/wmi-start-page))
+service.
+
+If telegraf is configured with a logfile and the plugin's configuration
+contains an invalid namespace, class, or property, an error is logged.

```

Comment:
Is this necessary to keep? I only asked as I was confused why this would print out if telegraf has a logfile configured and not just to stdout?

Reply:
Removed.